.changeset/odd-oranges-yell.md

```diff
@@ -0,0 +1,6 @@
+---
+"@digdir/designsystemet-css": patch
+"@digdir/designsystemet-react": patch
```

Comment:
Lets start getting used to using correct semvers, I think this would qualify as minor version bump?
```suggestion
"@digdir/designsystemet-css": minor
"@digdir/designsystemet-react": minor
```